Tests/StreamChatUITests/SnapshotTests/ChatMessageList/Attachments/ChatMessageVoiceRecordingAttachmentListView_Tests.swift

```diff
@@ -0,0 +1,142 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatTestTools
+@testable import StreamChatUI
+import UIKit
+import XCTest
+
+final class ChatMessageVoiceRecordingAttachmentListView_Tests: XCTestCase {
```

Comment:
Any reason why the waveform is not shown in the snapshot specs?

Reply:
This is because we haven't added any `waveformData`. No reason why not to have something there. Let me add some data.